tests/src/test/scala/scalaz/DisjunctionTest.scala

```diff
@@ -11,6 +11,7 @@ object DisjunctionTest extends SpecLite {
   checkAll(monoid.laws[Int \/ Int])
   checkAll(bindRec.laws[Int \/ ?])
   checkAll(monad.laws[Int \/ ?])
+  checkAll(applicativeError.laws[Int \/ ?, Int])
```

Comment:
if you make `MonadErrorLaws extends ApplicativeErrorLaws` you don't need this, right?